docs/packagedelivery.md

```diff
@@ -0,0 +1,11 @@
+# Package Delivery Example
+
+![package delivery sim](packagedeliverysim.gif)
+
+The system under test is the robot moving on the grid, its objective is to delivery all five packages to their goal location (shaded in the same color). The order of delivery does not matter to the robot. Finally, it needs to reach its target location on the left edge of the grid (orange).
```

Comment:
```suggestion
The system under test is the robot moving on the grid. Its objective is to delivery all five packages to their goal location (shaded in the same color). The order of delivery does not matter to the robot. Finally, it needs to reach its target location on the left edge of the grid (orange).
```
It is ungrammatical to join the sentences with "," here without a conjunction word like "and". Alternatively, as I suggest above, these can be split into 2 separate sentences.